docker/images/docker-spark/base/Dockerfile

```diff
@@ -41,5 +39,5 @@ COPY clueso /clueso
 RUN mkdir -p /clueso/heapdumps && mkdir -p /clueso/conf/
 COPY application.conf /clueso/conf/
 COPY conf /spark/conf/ 
-
+RUN echo 'networkaddress.cache.ttl=20' >> $JAVA_HOME/jre/lib/security/java.security
```

Comment:
this is the line that stops it from caching the wrong address?  Please add a comment as to why here. Also, why 20?

what about the blacklist/negative cache?

Reply:
Yeah so the default is to "forever cache" aka -1 and 20sec seemed like a reasonable time but can be changed. 0 might be never cache but I'm not certain of it.

For the blacklist/negative cache, the default is to cache the past 10seconds of failed lookups which is why I didn't add it but I can have it forever cache aka -1